packages/gatsby-theme-newrelic/src/components/SEO.js

```diff
@@ -38,6 +38,14 @@ const SEO = ({ title, location, children }) => {
       ? location.pathname
       : location.pathname.replace(new RegExp(`^\\/${locale.locale}`), '/');
 
+  const getSwiftypeSiteType = () => {
+    const nrSubDomain = /.*\.newrelic\.com/.test(location.hostname)
+      ? location.hostname.split('.')[0]
+      : 'demo';
+    const localeString = !locale.isDefault ? `-${locale.locale}` : '';
```

Comment:
[nit] I generally like to use positive connotation first. Typically its easier to think in the positive and helps your brain digest it faster (i.e. "if this... then that" vs "if not this... then that"). Would you be willing to swap these?

Feel free to leave it as-is if you feel strongly. Figured I'd point this out as its really helped me over the years understand conditional logic much quicker when reading through a codebase.